File: Java/Array-1/makeEnds/MakeEnds.java
package makeEnds;

public class MakeEnds {
    //Retorne um Array de "length" 2 que seja formado pelo primeiro e último elemento do Array dado, o Array dado deve
    //ter "length" de 1 ou mais.

    public int[] makeEnds(int[] nums) {
        return new int[]{nums[0],nums[nums.length-1]};
    }
}

File: Python/List-1/make_ends.py
"""
Dada uma lista numérica "nums", retorne outra que contenha o seu primeiro e último
elemento. A lista dado terá tamanho de pelo menos 1.
"""


def make_ends(nums):
    return nums[0:1] + nums[-1:]


print(make_ends([1, 2, 3]))
